### DISCUSSION TITLE
Suggestions for editing post comments.

### DISCUSSION THREAD

**Discussion by oklokl** [community/community#37046] [Discussions]
When I edit a post I made, the comment disappears.
I would like to keep the existing comments as they are.

And when writing README.md.
Inconvenience of having to run two spaces.
I want to jump one space at a time.
Enter.
Currently I have to jump through the lines of a post two lines at a time.
I am a beginner.

I can't speak English, so I'm uploading a video.
https://youtu.be/CLjzqdEwD7Q

And I'm greedy to add another suggestion.
Even for files with the same name. depending on upload time
I hope it's a duplicate upload. and
The url of the file is in a different form.
We suggest a service where files with the same name can be uploaded.

If the developer wants to load, the last uploaded form is loaded.

And I'd love to support these two. 
Download file..
`curl.exe --output`
`bitsadmin /transfer`

It is a pity that only powershell is supported now.

And I wish I could download the files individually.
on the web site.

Currently, you have to integrate it and get it as a zip.
Or you have to get it as a command.
Discomfort..

If I can get one post and one file if I can click one
I think it will be convenient.

**Comment by DenverCoder1** [comment]
> When I edit a post I made, the comment disappears. I would like to keep the existing comments as they are.

I'm not sure I quite understood, but it seems like you are referring to the "latest commit" message. This will change every time you make a commit since that is what the message is for. You can click the History icon to view previous commit messages.

> And when writing README.md.
> Inconvenience of having to run two spaces.
> I want to jump one space at a time.
> Enter.
> Currently I have to jump through the lines of a post two lines at a time.

Requiring extra space between lines is standard across nearly all types of markdown including GitHub's, there are however some other ways to force a line break (see: https://github.github.com/gfm/#hard-line-breaks)

One is to add two or more spaces at the end of the first line
![image](https://user-images.githubusercontent.com/20955511/197405651-dd2e81d2-b214-4d63-8b9c-0802b893c8eb.png)

Another is to put a backslash at the end
![image](https://user-images.githubusercontent.com/20955511/197405665-4d4cf845-9a3e-46af-a883-aa27b9cc4c4e.png)

Both of these will render with a line break in between.

If you want to start a new paragraph, there needs to be an empty line in between paragraphs.

The reason for this is because it allows for line wrapping at a certain number of characters within a single paragraph.


> And I wish I could download the files individually.

As a workaround, you can click "Raw" and then <kbd>Ctrl</kbd>+<kbd>S</kbd> to save. I do agree that it is a bit surprising that there doesn't seem to be a direct button for it.

**Comment by oklokl** [comment]
I am of the opinion that the existing function should be improved.
I hope that even beginners can easily access it.

The problem of disappearing comments is inconvenient when categorizing and viewing by external users.
It is also inconvenient to have to change it every time.
So I made a suggestion.

To summarize my words, just like we write on a blog,
I wish I could be comfortable.

This is my wish as a beginner.
:)

**Comment by ossanna16** [comment]
Hi there @oklokl and welcome to our community 😄. Thanks so much for asking a great question!
